clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2230,9 +2216,24 @@ class SyclKernelDeclCreator : public SyclKernelFieldHandler {
     // device in memory allocations
     if (AS != LangAS::sycl_global_device && AS != LangAS::sycl_global_host)
       Quals.setAddressSpace(LangAS::sycl_global);
-    PointeeTy = SemaRef.getASTContext().getQualifiedType(
-        PointeeTy.getUnqualifiedType(), Quals);
-    QualType ModTy = SemaRef.getASTContext().getPointerType(PointeeTy);
+    // Handle VLAs if this is a pointer to one
```

Comment:
```suggestion
    // Handle VLAs if this is a pointer to one.
```
Same suggestion to add trailing punctuation applies elsewhere in the file.